### ISSUE TITLE
IP Week 5 local master VS remote master

### ISSUE THREAD

**Issue by gavin331** [nus-cs2103-AY2324S2/forum#273]
In part 3 of week 5 Individual Project tasks, it is mentioned that after merging 1 of the PRs on github, the master branch of the local repo will fall behind that of the fork. However, after I merged 1 of the PRs on github, and then I used the git status command in the local master branch,  it still says the local master branch is up to date with origin master although the local master branch do not have the latest changes compared to the remote master branch. 
Why does it say that the local master is up to date with remote master if the local master do not have the latest changes? The local master branch only received the latest changes after I 'git pull origin master'.

**Comment by dillontkh** [comment]
I found [this stackoverflow answer](https://stackoverflow.com/questions/37694515/git-status-vs-fetch-or-pull-how-to-know-what-up-to-date-means-without-chan) quite informative.

TL;DR:
The flow of information from remote to your local branch is something like this
`remote master (repo) --> origin/master (local tracking branch) --> master (local branch)`

Your local `master` branch was current with the `origin/master` local tracking branch, but the `origin/master` local tracking branch was not up to date with the remote `master`.

**Comment by drustanyjt** [comment]
@gavin331 to add on, you can use `git fetch` to see if there are changes on the remote. If you did
```
git fetch
git status
```
You should then see that your 'master' branch is behind 'origin/master'

**Comment by damithc** [comment]
> @gavin331 to add on, you can use `git fetch` to see if there are changes on the remote. If you did
> 
> ```
> git fetch
> git status
> ```
> 
> You should then see that your 'master' branch is behind 'origin/master'

Good info @drustanyjt 
I've added a mention of that in the course website.

**Comment by gavin331** [comment]
Ok thanks everyone!
